_integration-test/conftest.py

```diff
@@ -1,58 +1,22 @@
 import os
+from os.path import join
 import subprocess
-import time
 
-import httpx
 import pytest
 
 SENTRY_CONFIG_PY = "sentry/sentry.conf.py"
 SENTRY_TEST_HOST = os.getenv("SENTRY_TEST_HOST", "http://localhost:9000")
 TEST_USER = "test@example.com"
 TEST_PASS = "test123TEST"
-TIMEOUT_SECONDS = 60
-
-
-def pytest_addoption(parser):
-    parser.addoption("--customizations", default="disabled")
-
 
 @pytest.fixture(scope="session", autouse=True)
 def configure_self_hosted_environment(request):
     subprocess.run(
-        ["docker", "compose", "--ansi", "never", "up", "-d"],
+        ["docker", "compose", "--ansi", "never", "up", "--wait"],
```

Comment:
Is this needed anymore? Seeing https://github.com/getsentry/self-hosted/blob/63334cbcc23132e4d6832809ed399c75f48e4683/action.yaml#L135

Reply:
I'm not sure but I'll try removing it and see what happens.